packages/components/psammead-podcast-promo/src/__snapshots__/index.test.jsx.snap

```diff
@@ -517,65 +513,56 @@ exports[`Podcast Promo Examples basic example 1`] = `
             width="100%"
           />
         </div>
-      </figure>
+      </div>
       <div
-        class="emotion-26 emotion-27"
+        class="emotion-24 emotion-25"
       >
         <h3
-          class="emotion-16 emotion-17"
+          class="emotion-14 emotion-15"
         >
           <a
-            class="emotion-14 emotion-15"
+            class="emotion-12 emotion-13"
             href="https://www.bbc.com/russian/media-47937790"
           >
             <span
               class="podcast-promo--hover podcast-promo--focus podcast-promo--visited"
             >
               Что это было?
             </span>
-            <span
-              class="emotion-12 emotion-13"
-            >
-              , Episodes
-            </span>
           </a>
         </h3>
         <p
-          class="emotion-18 emotion-19"
+          class="emotion-16 emotion-17"
         >
           Мы быстро, просто и понятно объясняем, что случилось, почему это важно и что будет дальше. Никаких ненужных подробностей и передергиваний - только факты и взвешенная аналитика.
         </p>
         <span
```

Comment:
I would prefer to see this inside the p tag as it's associated with that content

Reply:
Sure, that makes sense - done!